.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Feature Status
+**READY/IN DEVELOPMENT**
+
+## Related PRs
+List related PRs against other branches e.g. for backporting features/bugfixes
+to previous release branches:
+
+Repo/Branch | PR
+------ | ------
+some_other_PR | [link]()
+
+
+## Todos
+- [ ] Tested and working on development environment
+- [ ] Unit tests (if appropriate)
```

Comment:
Add a line: Added or updated all related documentation

Reply:
missed that and added *Tested and working on development environment* twice. will fix